editorconfig.go

```diff
@@ -42,6 +42,13 @@ const (
 	CharsetUTF8BOM = "utf-8 bom"
 )
 
+// Limits for section name, properties, and values.
+const (
+	MaxPropertyLength = 50
+	MaxSectionLength  = 4096
+	MaxValueLength    = 255
```

Comment:
Just a question we should make here (no strong opinion on this, though): should these constants really be exported? Are they useful for the user?

Reply:
I don't see any problems exporting those *magic numbers*. I followed the way it's been done above.